fe1-web/src/features/notification/screens/NotificationScreen.tsx

```diff
@@ -0,0 +1,153 @@
+    const read: NotificationItem[] = [];
+    const unread: NotificationItem[] = [];
+    for (const notification of notifications) {
+      if (notification.hasBeenRead) {
+        read.push({ ...notification, isLastItem: false });
+      } else {
+        unread.push({ ...notification, isLastItem: false });
+      }
+    }
+
+    // sort in descending order, i.e. newest/latest first
+    read.sort((a, b) => b.timestamp - a.timestamp);
+    unread.sort((a, b) => b.timestamp - a.timestamp);
+
+    const items: (ListSeparatorItem | NotificationItem)[] = [];
+    if (unread.length > 0) {
+      // set isLastItem properties
+      unread[unread.length - 1].isLastItem = true;
+
+      items.push({ title: 'Notifications' } as ListSeparatorItem, ...unread);
+    }
+
+    if (read.length > 0) {
+      // set isLastItem properties
+      read[read.length - 1].isLastItem = true;
+
+      items.push({ title: 'Read Notifications' } as ListSeparatorItem, ...read);
+    }
+
+    return items;
+  }, [notifications]);
+
+  const notificationTypes = NotificationHooks.useNotificationTypes();
+
+  const onClearNotifications = () => {
+    // call custom delete function on all notifications
+    for (const notification of notifications) {
+      const deleteFn = notificationTypes.find((t) => t.isOfType(notification))?.delete;
+
+      // if a delete function was provided for this type, then call it
+      if (deleteFn) {
+        deleteFn(notification);
+      }
+    }
+    // remove notifications from the notification reducer
+    dispatch(
+      discardNotifications({
+        laoId: laoId.valueOf(),
+        notificationIds: notifications.map((n) => n.id),
+      }),
+    );
+  };
+
+  return (
+    <ScreenWrapper>
+      <FlatList
+        data={notificationData}
+        keyExtractor={(item) => ('id' in item ? item.id.toString() : item.title)}
+        renderItem={({ item }) => {
+          if ('id' in item) {
+            // notification
+            return (
+              <TouchableOpacity
+                onPress={() =>
+                  navigation.navigate<'Notification'>(
+                    STRINGS.notification_navigation_tab_single_notification,
+                    { notificationId: item.id },
+                  )
+                }>
+                <View
+                  style={
+                    item.isLastItem
+                      ? [
+                          NotificationScreenStyles.notificationItem,
+                          NotificationScreenStyles.lastNotificationItem,
+                        ]
+                      : NotificationScreenStyles.notificationItem
+                  }>
+                  <Text>{item.title}</Text>
+                </View>
+              </TouchableOpacity>
+            );
+          }
+          // separator
+          return <Text style={Typography.important as TextStyle}>{item.title}</Text>;
+        }}
+      />
+      <Button title="Clear notifications" onPress={onClearNotifications} />
```

Comment:
All strings that are displayed in the UI shall be defined in `strings` to allow easier translation.